### ISSUE TITLE
Migrated forms should use the default form template

### ISSUE THREAD

**Issue by jdsdev** [verbb/formie#572]
**Description**

Migrated forms are assigned the "Default Formie Template", it would be preferable if they were assigned the "Default Form Template", or there were an option to select the preferred template prior to migration.

**Additional info**
- Plugin version: 1.4.15

**Comment by engram-design** [comment]
They should certainly respect the default form template. Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.16](https://github.com/verbb/formie/releases/tag/1.4.16)
